### DISCUSSION TITLE
Please help me with my GitHub Education application

### DISCUSSION THREAD

**Discussion by ghost** [community/community#144182] [GitHub Education]
### Hi everyone,

I am a student living in South Korea, and apart from My school records(in korean, '생활기록부'), which are a copy of an official document, I do not have a student ID or an admission letter. How should I proceed?

**Comment by samuel-404** [comment]
To apply for GitHub Education without a student ID or admission letter, here’s what you can do:

Use your school records (생활기록부): These are official documents, so they can serve as proof of enrollment. Just make sure the copy is clear and shows your full name, school name, and enrollment dates.

Translate your records into English: Since the records are in Korean, it’s a good idea to provide an English translation. You can do this yourself, or use a translation service.

Explain your situation: In the application, you can mention that you don’t have a student ID or admission letter, but you’re submitting your official school records as proof of enrollment. Let them know you’ve also included a translation.

Add any other supporting materials: If you have anything else that could help (like an email from your school or an enrollment confirmation), include it.

If you’re unsure, you can always contact GitHub Education support for advice. They’ll be able to guide you through the process.

**Reply by ghost** [reply]
did u use gpt? how did u write korean.??

**Comment by ignaciostellino** [comment]
@samuel-404 are you able to help me with this one https://github.com/orgs/community/discussions/143658 ? Thanks

**Comment by lllHunterlllSV** [comment]
**Hello! 👋**

Thank you for sharing your situation. I understand that it can be tricky when the available documentation doesn’t match the typical requirements for GitHub Education. Let’s work through some potential solutions:

- 📝 **Submit Your School Records**: Since you have your official school records (생활기록부), this can often serve as a valid form of documentation, especially if it clearly shows your name, school details, and student status. Even though it’s in Korean, it can still be accepted, as long as the document is official and legible.

    - 💬 **Note on Language**: If you’re concerned about the language barrier, it’s a good idea to include a brief explanation in English alongside the document, explaining that the records are official school documents in Korean. This can help reviewers understand the document’s context more easily.

- 🏫 **Explain Your Situation**: In your GitHub Education application, be sure to provide an explanation that clarifies your current situation. Let them know that you don’t have a student ID or an admission letter, but that you are currently enrolled and provide a scan or photo of your official school records. 

- 📧 **Reach Out to GitHub Support**: If you're still unsure whether your documentation will be accepted, you can contact [GitHub Education Support](https://support.github.com/) and explain your situation. They may be able to give you specific instructions or offer an alternative way to submit your documentation.

- 📄 **Other Documentation**: If possible, try to provide additional supporting documents, like a recent transcript, proof of enrollment, or any other official communication from your school that confirms your student status.

I hope this helps you navigate the process, and I wish you the best of luck with your GitHub Education application! 😊

**Reply by ignaciostellino** [reply]
@lllHunterlllSV 
1) are you aware that is impossible to contact support as all the options said that better to create a discussion under Github Education?
2) are you aware that the form only allows to upload a live photo and only with the front camera? 
How we are suppose to take a clear picture that include all the require information with the front camera?
